packages/blade/src/components/Box/Box.tsx

```diff
@@ -1,10 +1,42 @@
+import React from 'react';
 import BaseBox from './BaseBox';
-import type { BoxProps } from './BaseBox/types';
+import type { BoxProps, MakeValueResponsive } from './BaseBox/types';
+import type { KeysRequired } from '~src/_helpers/types';
+import { metaAttribute, MetaConstants } from '~utils';
+
+const isValidBackgroundColorString = (stringBackgroundColorValue: string): void => {
+  if (!stringBackgroundColorValue.startsWith('surface.background')) {
```

Comment:
yup right. Should be fine right? since we get autocomplete. we're doing lodash get when we are actually adding backgroundColor inside `BaseBox`. So passing `surface.backgorund.jazz` just won't work.

runtime check might be faster with startsWith? 🤔

Reply:
yep that should work since we have boundary at BaseBox 👍